moped-database/migrations/1694809881268_moped_project_updated_at_index/up.sql

```diff
@@ -0,0 +1,2 @@
+CREATE INDEX moped_project_updated_at_idx ON moped_project (updated_at);
+CREATE INDEX moped_project_name_idx on moped_project (project_name);
```

Comment:
`updated_at` is the default column we use for sorting in the editor. i also figured it was wise to index on project name since we're so often searching against it.